app/upgrades/v16/upgrades.go

```diff
@@ -49,18 +49,6 @@ var (
 	USDCIBCDenom = "ibc/D189335C6E4A68B513C10AB227BF1C1D38C746766278BA3EEB4FB14124F1D858"
 	SpreadFactor = osmomath.MustNewDecFromStr("0.002")
 
-	// authorized_quote_denoms quote assets that can be used as token1
-	// when creating a pool. We limit the quote assets to a small set
-	// for the purposes of having convenient price increments stemming
-	// from tick to price conversion. These increments are in a human
-	// understandeable magnitude only for token1 as a quote.
-	authorizedQuoteDenoms []string = []string{
-		appparams.BaseCoinUnit,
-		ATOMIBCDenom,
-		DAIIBCDenom,
-		USDCIBCDenom,
-	}
```

Comment:
Can you comment out these items that are in the upgrades folder instead of deleting them? We like to keep the upgrades as close to how they were when they occurred, for easily looking back on what occurred during that upgrade.

See https://github.com/deividaspetraitis/osmosis/blob/3ff40a02ec9a241034e214e00d01535ded6a46a4/app/upgrades/v15/upgrade_test.go#L208-L209 as an example.

Reply:
Of course, please see updated changes!